[DOCS-6135] Link to Deployment and Architecture docs (#19687)
DataDog/documentation: content/en/observability_pipelines/_index.md

1. [Set up the Observability Pipelines Worker][1].
2. [Create pipelines to collect, transform and route your data][2].

> 3. Deep dive into recommendations for deploying and scaling the Observability Pipelines Worker:

Author: ```suggestion
3. Deep dive into recommendations for deploying and scaling multipe Observability Pipelines Workers:
```

> 3. Deep dive into recommendations for deploying and scaling multipe Observability Pipelines Workers:

Reviewer: ```suggestion
3. Deep dive into recommendations for deploying and scaling multiple Observability Pipelines Workers:
```

Spelling.

Reviewer: But this also reads kind of funny to me. On one hand I think `Dive deep` sounds better to me, but I am not sure if that is grammatically correct or not. But maybe side step it altogether with an alternative phrasing such as:

"Discover how to deploy Observability Pipelines at production scale"

Thoughts @barieom ?

Author: That suggestion sounds good to me!

Reviewer 2: Agree, I like your suggestion @ckelner

    - See [Deployment Design and Principles][3] for information on what to consider when designing your Observability Pipelines architecture.

>     - See [Best Practices for OPW Aggregator Architecture][4] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.

Reviewer: This repeats itself a bit (or seems wordy to me), what about: "See [Best Practices for OPW Aggregator Architecture][4] for recommended Observability Pipelines architecture, optimized for scaling."

Just brainstorming, but it is ok as-is too.

Author: Or we could just cut it down to:

> See [Best Practices for OPW Aggregator Architecture][4].

The title does explain itself :).

>     - See [Best Practices for OPW Aggregator Architecture][4] for details on the recommended Observability Pipelines aggregator architecture, which is optimized for scaling.

Author: ```suggestion
    - See [Best Practices for OP Worker Aggregator Architecture][4].
```
